types/priv_validator_test.go

```diff
@@ -99,7 +99,6 @@ func TestSignVote(t *testing.T) {
 	privVal := GenPrivValidatorFS(tempFilePath)
 
 	block1 := BlockID{[]byte{1, 2, 3}, PartSetHeader{}}
-	block2 := BlockID{[]byte{3, 2, 1}, PartSetHeader{}}
```

Comment:
why are we removing ?

Reply:
do you mean why remove the test case? because we removed the check and now test fails with "expected err, but got nil"